test/helpers/testSetup.js

```diff
@@ -64,9 +71,11 @@ const setupContracts = async () => {
     random,
     schellingCoin,
     stakeManager,
+    stakeRegulator,
```

Comment:
Add this at the end with stakedToken

Reply:
Hmm I think its better with stakeManager as both are related
And as of now I am not sure if we are going with this naming itself.
Lets see this once we all agree that stakeRegulator and stakeManager are final names.